packages/s3/adapter.ts

```diff
@@ -0,0 +1,72 @@
+import { StorageAdapter } from 'https://lib.deno.dev/x/grammy@1.x/mod.ts';
+import { S3Client } from 'jsr:@bradenmacdonald/s3-lite-client@0.7.4';
+export { S3Client } from 'jsr:@bradenmacdonald/s3-lite-client@0.7.4';
```

Comment:
Import it from deps file, so then when we'll nodefy this adapter dependencies will be used from deps.node.ts.

Reply:
makes sense

Reply:
:heavy_check_mark: